notebooks/Read_and_plot_LST_Data.ipynb

```diff
@@ -812,7 +812,11 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
+<<<<<<< HEAD
    "version": "3.7.3"
+=======
+   "version": "3.6.5"
+>>>>>>> 71399bf6e2af2638cd48a61813380c1583a879ea
```

Comment:
Please clean-up this.
(in this case I don't think this notebook needs to be included)